editor/src/components/inspector/sections/component-section/data-selector-modal.tsx

```diff
@@ -517,7 +536,6 @@ export const DataSelectorModal = React.memo(
                       {primitiveVars.map((variable) => (
                         <CartoucheUI
                           key={variable.valuePath.toString()}
-                          tooltip={variableNameFromPath(variable)}
```

Comment:
I removed the tooltips because they just repeated the label and made the screenshots look bad